.pylintrc

```diff
@@ -0,0 +1,37 @@
+[MAIN]
+
+# Specify a score threshold under which the program will exit with error.
+fail-under=10
+
+[BASIC]
+
+# Minimum line length for functions/classes that require docstrings, shorter
+# ones are exempt.
+docstring-min-length=100
```

Comment:
As far as I understand, clang_format uses 150 https://github.com/jamulussoftware/jamulus/blob/main/.clang-format#L40 I think we should remain consistent. Also for the line length.

Reply:
This is the number of _rows_, not columns.  See [an explanation here](https://stackoverflow.com/questions/26889188/is-it-possible-to-disable-pylint-missing-docstring-error-for-short-functions-and) or [its docs](https://pylint.pycqa.org/en/latest/user_guide/configuration/all-options.html#docstring-min-length)

Methods / functions under this limit _don't_ need comments. This is to avoid creating too much churn here, and I'm not really in a position to comment other people's methods (currently at least!)

Reply:
Ah ok. 100 seems quite long, but we don't even require commenting functions anywhere. So it's ok.